lib/pipeline.js

```diff
@@ -88,6 +88,23 @@ class PipelineModel extends BaseModel {
 
         return result.toLowerCase();
     }
+
+    /**
+     * Given a pipeline id, fetch the build admin
+     * @method getAdmin
+     * @param  {String}       pipelineId    Identifier for the Pipeline
+     * @param  {Function}     callback      fn(error, admin) where admin is the admin of the pipeline
+    */
+   // TODO: Retry if admin access was revoked
+    getAdmin(pipelineId, callback) {
```

Comment:
This funciton should only get ONE admin that is valid.  If admin access was revoked, then it should retry until we get one that is valid. And at the same time, remove that person from the admin list. Whoever consumes this shouldn't have to do the retrying and removing. Seems more like the responsibility of the pipeline.

Also, when would ever need to get the list of admins? At least I don't see that use case right now.